wagtail/admin/views/pages/edit.py

```diff
@@ -75,6 +75,15 @@ def add_save_confirmation_message(self):
                 "page_title": self.page.get_admin_display_title()
             }
 
+        if self.updated_by_another_user:
+            overwrite_warning_message = _(
+                "Page Revision %(initial_page_revision)s overwriting changes by another user in %(overwritten_page_revision)s."
+            ) % {
+                "initial_page_revision": self.request.POST.get("page_revision_id"),
+                "overwritten_page_revision": self.page.get_latest_revision().id,
```

Comment:
I think the wording here is quite ambiguous because it sounds like `overwritten_page_revision` was the one the other used saved. And the `initial_page_revision` is not really the one that's being saved now. I'd simply suggest to the user to review the version history instead, without mentioning revision IDs. Or maybe list only the revision IDs that were in between these too (unlikely, but could be more than one).